src/%IPM/Main.cls

```diff
@@ -156,75 +188,111 @@ repo -n LocalFiles -delete
+<modifier name="disable" description="Disables package manager web services in this namespace." />
+<modifier name="info" description="Shows configuration details for the current namespace as a remote repository." />
+<modifier name="copy-from" value="true" description="Copies repository configuration from a different (specified) namespace" />
 <!-- General/shared modifiers -->
 <modifier name="name" aliases="n" value="true" description="Namespace-unique name for the module" />
 <modifier name="delete" description="Deletes the current namespace's reference to the named repository" />
 <modifier name="publish" value="true" valueList="0,1" description="When configuring a remote repository, specifies that publishing of packages to the repository is allowed. When configuring the current namespace with -enable, specifies that publishing is also enabled. "/>
-
 <!-- General properties -->
 <modifier name="snapshots" aliases="s" value="true" valueList="0,1" description="For any repository, specifies that it should be used to look for snapshot builds (i.e., those with a semantic version ending in '+snapshot', indicating a 'latest' build of a particular version)." />
 <modifier name="prereleases" aliases="pre" value="true" valueList="0,1" description="For any repository, specifies that it should be used to look for prerelease software" />
 <modifier name="read-only" aliases="ro" value="true" valueList="0,1" description="For any repository, specifies that modules installed from it are ALWAYS installed in non-developer mode, effectively making them read only" />
-
+<!-- Repository types  -->
+<modifier name="local" aliases="l" description="Create/update the local namespace cache" />
 <modifier name="type" value="true" aliases="t" description="Subclass of %IPM.Repo.Definition to create/modify/delete or implementation of %IPM.Repo.IPackageService or %IPM.Repo.IPublishService to enable/disable" />
+<!-- Modifiers specific to -enable -->
+<modifier name="app" value="true" description="When enabling a namespace to serve as a remote repository, specifies the web application to allow web service access." />
 
 </command>
 
 <command name="load" dataPrefix="D">
-<description>Loads a module from the specified directory or archive into the current namespace. Dependencies are also loaded automatically, provided that they can be found in repositories configured with the 'repo' command.</description>
+<summary>
+Loads a module from the specified directory into the current namespace.
+</summary>
+<description>
+Loads a module from the specified directory into the current namespace.
+Dependencies are also loaded automatically, provided that they can be found in
+repositories configured with the 'repo' command.
+</description>
+
+<!-- Examples -->
 <example description="Loads the module described in C:\module\root\path\module.xml">
 load C:\module\root\path\
-load C:\module\root\path\module-0.0.1.tgz
```

Comment:
Does it not work with a tarball? If not, then remove, but if so, should keep it here

Reply:
As far as from my testing, it will work with tarball (only tgz, and will need to have .xml file at the very first level of the tarball). I will update the comments a little